frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -86,6 +86,7 @@ const Map: React.FC<MapProps> = ({
   const lastZoom = useSelector<RootState, number>(state => state.mapViewZoom) ?? zoom;
   useEffect(() => {
     dispatch(resetMapViewZoom());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
A tech-debt suggestion for later: I remember excluding this rule in our `package.json` to avoid having to add it everywhere explicitly. I followed `create-react-app` documentation but something must not be working. You can see the rule turned off on https://github.com/bcgov/PIMS/blob/dev/frontend/package.json#L92

Reply:
I worry we’re doing something wrong because the warming is everywhere.

Reply:
yeah, something might be wrong somewhere. I doubt we will have the time to debug this given all we want to accomplish. maybe next sprint